src/Sections/EntitiesSection/EntitiesManager.ts

```diff
@@ -68,6 +71,17 @@ export default abstract class EntitiesManager implements DxfInterface {
     return entity
   }
 
+  addAttrib(firstAlignmentPoint: vec3_t, height: number, tag: string, value: string, ownerInsert: Insert, options?: TextOptions) {
+    ownerInsert.attributesFollowFlag = 1
+    const attrib = this.addEntity(new Attrib(firstAlignmentPoint, height, tag, value, options))
+    this.addEntity(new SeqEnd(ownerInsert.handle))
```

Comment:
no need to pass the owner like that you can change it. You are basically doing this
```ts
const seqEnd = this.addEntity(new SeqEnd())
seqEnd.ownerBlockRecord = ownerInsert.handle
```